### ISSUE TITLE
[APP SUBMITTED]: saveSearch() got an unexpected keyword argument 'torrent_download_dir_deluge'

### ISSUE THREAD

**Issue by pejub** [SickChill/SickChill#5500]
### INFO
Python Version: **2.7.15rc1 (default, Nov 12 2018, 14:31:15) [GCC 7.3.0]**
Operating System: **Linux-4.15.0-48-generic-x86_64-with-Ubuntu-18.04-bionic**
Locale: UTF-8
Branch: **master**
Commit: SickChill/SickChill@34fdd20ee3f8ec8799b7577cfdbdc9cf7bb5f73c
No Log available with ERRORS:
### ERROR
```
ThreadPoolExecutor-0_1 :: [34fdd20] Failed doing webui callback: Traceback (most recent call last):
  File "/opt/sickrage/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
TypeError: saveSearch() got an unexpected keyword argument 'torrent_download_dir_deluge'

```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by CraigInBrisbane** [comment]
Can confirm same problem. Unable to save my search settings. Chrome debug returns 500.

<title>saveSearch() got an unexpected keyword argument 'torrent_download_dir_deluge'</title>
                                 <body>
                                    <h2>Error</h2>
                                    <p>saveSearch() got an unexpected keyword argument 'torrent_download_dir_deluge'</p>
                                    <h2>Traceback</h2>
                                    <p>Traceback (most recent call last):
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/tornado/web.py", line 1511, in _execute
    result = yield result
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/tornado/gen.py", line 1055, in run
    value = future.result()
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/webserve.py", line 265, in get
    results = yield self.async_call(method)
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/tornado/gen.py", line 1055, in run
    value = future.result()
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/concurrent/futures/_base.py", line 455, in result
    return self.__get_result()
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/concurrent/futures/thread.py", line 63, in run
    result = self.fn(*self.args, **self.kwargs)
<br>  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
<br>TypeError: saveSearch() got an unexpected keyword argument 'torrent_download_dir_deluge'
<br></p>
                                    <h2>Request Info</h2>
                                    <p><strong>body</strong>: download_propers=on&check_propers_interval=daily&backlog_days=7&backlog_frequency=720&dailysearch_frequency=40&usenet_retention=500&ignore_words=german%2Cfrench%2Ccore2hd%2Cdutch%2Cswedish%2Creenc%2CMrLss&require_words=&trackers_list=udp%3A%2F%2Fcoppersurfer.tk%3A6969%2Fannounce%2Cudp%3A%2F%2Fopen.demonii.com%3A1337%2Cudp%3A%2F%2Fexodus.desync.com%3A6969%2Cudp%3A%2F%2F9.rarbg.me%3A2710%2Fannounce%2Cudp%3A%2F%2Fglotorrents.pw%3A6969%2Fannounce%2Cudp%3A%2F%2Ftracker.openbittorrent.com%3A80%2Fannounce%2Cudp%3A%2F%2F9.rarbg.to%3A2710%2Fannounce&ignored_subs_list=dk%2Cfin%2Cheb%2Ckor%2Cnor%2Cnordic%2Cpl%2Cswe&allow_high_priority=on&use_nzbs=on&nzb_method=sabnzbd&sab_host=192.168.1.5%3A8123&sab_username=&sab_password=&sab_apikey=8a24d4e3c2c72396058ea82a6f5f33ab&sab_category=tv&sab_category_backlog=tv&sab_category_anime=anime&sab_category_anime_backlog=anime&nzb_dir=&nzbget_host=&nzbget_username=nzbget&nzbget_password=hidden_value&nzbget_category=tv&nzbget_category_backlog=tv&nzbget_category_anime=anime&nzbget_category_anime_backlog=anime&nzbget_priority=100&syno_dsm_host=&syno_dsm_user=&syno_dsm_pass=&syno_dsm_path=&use_torrents=on&torrent_method=utorrent&torrent_dir=&torrent_host=http%3A%2F%2Flocalhost%3A8000&torrent_rpcurl=transmission&torrent_auth_type=none&torrent_username=3424&torrent_password=3243242&torrent_label=324324&torrent_label_anime=3243&torrent_download_dir_deluge=&torrent_complete_dir_deluge=&torrent_path=&torrent_seed_time=-1<br><strong>files</strong>: {}<br><strong>protocol</strong>: http<br><strong>connection</strong>: <tornado.http1connection.HTTP1Connection object at 0x7f6a3f5bd0><br><strong>body_arguments</strong>: {'usenet_retention': ['500'], 'download_propers': ['on'], 'use_nzbs': ['on'], 'backlog_frequency': ['720'], 'syno_dsm_host': [''], 'nzbget_category': ['tv'], 'sab_host': ['192.168.1.5:8123'], 'sab_category_anime': ['anime'], 'sab_username': [''], 'allow_high_priority': ['on'], 'sab_apikey': ['8a24d4e3c2c72396058ea82a6f5f33ab'], 'nzbget_category_backlog': ['tv'], 'trackers_list': ['udp://coppersurfer.tk:6969/announce,udp://open.demonii.com:1337,udp://exodus.desync.com:6969,udp://9.rarbg.me:2710/announce,udp://glotorrents.pw:6969/announce,udp://tracker.openbittorrent.com:80/announce,udp://9.rarbg.to:2710/announce'], 'sab_category': ['tv'], 'dailysearch_frequency': ['40'], 'nzbget_category_anime': ['anime'], 'ignore_words': ['german,french,core2hd,dutch,swedish,reenc,MrLss'], 'torrent_label': ['324324'], 'syno_dsm_pass': [''], 'torrent_seed_time': ['-1'], 'check_propers_interval': ['daily'], 'torrent_host': ['http://localhost:8000'], 'nzb_method': ['sabnzbd'], 'sab_category_backlog': ['tv'], 'require_words': [''], 'nzb_dir': [''], 'nzbget_username': ['nzbget'], 'sab_category_anime_backlog': ['anime'], 'torrent_label_anime': ['3243'], 'nzbget_category_anime_backlog': ['anime'], 'sab_password': [''], 'torrent_download_dir_deluge': [''], 'torrent_dir': [''], 'torrent_path': [''], 'torrent_auth_type': ['none'], 'nzbget_priority': ['100'], 'torrent_method': ['utorrent'], 'ignored_subs_list': ['dk,fin,heb,kor,nor,nordic,pl,swe'], 'torrent_rpcurl': ['transmission'], 'torrent_complete_dir_deluge': [''], 'nzbget_host': [''], 'torrent_username': ['3424'], 'syno_dsm_user': [''], 'backlog_days': ['7'], 'nzbget_password': ['hidden_value'], 'torrent_password': ['3243242'], 'syno_dsm_path': [''], 'use_torrents': ['on']}<br><strong>uri</strong>: /config/search/saveSearch<br><strong>query_arguments</strong>: {}<br><strong>_start_time</strong>: 1558845999.76<br><strong>headers</strong>: Origin: http://192.168.1.5:8083
Content-Length: 1460
Accept-Language: en,en-US;q=0.9,ko;q=0.8,zh-CN;q=0.7,zh;q=0.6,pt;q=0.5,la;q=0.4
Accept-Encoding: gzip, deflate
Connection: keep-alive
Accept: */*
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36
Host: 192.168.1.5:8083
X-Requested-With: XMLHttpRequest
Cookie: smid=U6NTvmJ12ievSR4BewF9LJ6vTxpFBzZ6L8qrxoPMOwb8p98sWh0oVml8DSI6cWeNHanSab77HyRnXApoh3FRKw; stay_login=1; _pk_id.1.dfbd=1db498df1237130d.1552804498.2.1553514710.1553514509.; login_cookie=3daff1f496df8022e89ab1172d3a04c08488009c; login_salt=448; id=B40aAjqgP1hBM18B0PUN167600; user="2|1:0|10:1558786932|4:user|44:NzhkMjIyYTY1MDY4NDNhMWE3YjA1YjkyMGUyYTRiZDA=|f8df570b20ab2cfd6919db2f8b1d147d6d53f342284092b97dcc977ba8c1234d"; snatched_view=list; soon_view=thumb; suggest_view=thumb; late_view=list; wanted_view=thumb; blu_ray_com_new_releases_view=thumb; imdb_box_office_view=thumb; advanced_toggle_checked=1; manage_view=thumb; sickchill_user="2|1:0|10:1558830835|14:sickchill_user|44:ZTM2ZjJiOWRkYmMzMmZkYzM0MDE2MDQyM2ExMWIwOGU=|7a545ade6712a4bdaf1e9574377138b648f349aa2bc578a4563b67b44748e3b6"
Referer: http://192.168.1.5:8083/config/search/
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
<br><strong>host</strong>: 192.168.1.5:8083<br><strong>version</strong>: HTTP/1.1<br><strong>server_connection</strong>: <tornado.http1connection.HTTP1ServerConnection object at 0x7f9c0905d0><br><strong>host_name</strong>: 192.168.1.5<br><strong>_finish_time</strong>: None<br><strong>query</strong>: <br><strong>arguments</strong>: {'usenet_retention': '500', 'download_propers': 'on', 'use_nzbs': 'on', 'backlog_frequency': '720', 'syno_dsm_host': '', 'nzbget_category': 'tv', 'sab_host': '192.168.1.5:8123', 'sab_category_anime': 'anime', 'sab_username': '', 'allow_high_priority': 'on', 'sab_apikey': '8a24d4e3c2c72396058ea82a6f5f33ab', 'nzbget_category_backlog': 'tv', 'trackers_list': 'udp://coppersurfer.tk:6969/announce,udp://open.demonii.com:1337,udp://exodus.desync.com:6969,udp://9.rarbg.me:2710/announce,udp://glotorrents.pw:6969/announce,udp://tracker.openbittorrent.com:80/announce,udp://9.rarbg.to:2710/announce', 'sab_category': 'tv', 'dailysearch_frequency': '40', 'backlog_days': '7', 'nzbget_category_anime': 'anime', 'ignore_words': 'german,french,core2hd,dutch,swedish,reenc,MrLss', 'syno_dsm_pass': '', 'torrent_seed_time': '-1', 'check_propers_interval': 'daily', 'torrent_host': 'http://localhost:8000', 'nzb_method': 'sabnzbd', 'sab_category_backlog': 'tv', 'require_words': '', 'nzb_dir': '', 'nzbget_username': 'nzbget', 'sab_category_anime_backlog': 'anime', 'torrent_label_anime': '3243', 'nzbget_category_anime_backlog': 'anime', 'sab_password': '', 'torrent_download_dir_deluge': '', 'torrent_dir': '', 'torrent_path': '', 'torrent_auth_type': 'none', 'torrent_method': 'utorrent', 'ignored_subs_list': 'dk,fin,heb,kor,nor,nordic,pl,swe', 'torrent_rpcurl': 'transmission', 'torrent_complete_dir_deluge': '', 'use_torrents': 'on', 'nzbget_host': '', 'torrent_username': '3424', 'syno_dsm_user': '', 'torrent_label': '324324', 'nzbget_password': 'hidden_value', 'torrent_password': '3243242', 'syno_dsm_path': '', 'nzbget_priority': '100'}<br><strong>path</strong>: /config/search/saveSearch<br><strong>_cookies</strong>: Set-Cookie: _pk_id.1.dfbd=1db498df1237130d.1552804498.2.1553514710.1553514509.
Set-Cookie: advanced_toggle_checked=1
Set-Cookie: blu_ray_com_new_releases_view=thumb
Set-Cookie: id=B40aAjqgP1hBM18B0PUN167600
Set-Cookie: imdb_box_office_view=thumb
Set-Cookie: late_view=list
Set-Cookie: login_cookie=3daff1f496df8022e89ab1172d3a04c08488009c
Set-Cookie: login_salt=448
Set-Cookie: manage_view=thumb
Set-Cookie: sickchill_user="2|1:0|10:1558830835|14:sickchill_user|44:ZTM2ZjJiOWRkYmMzMmZkYzM0MDE2MDQyM2ExMWIwOGU=|7a545ade6712a4bdaf1e9574377138b648f349aa2bc578a4563b67b44748e3b6"
Set-Cookie: smid=U6NTvmJ12ievSR4BewF9LJ6vTxpFBzZ6L8qrxoPMOwb8p98sWh0oVml8DSI6cWeNHanSab77HyRnXApoh3FRKw
Set-Cookie: snatched_view=list
Set-Cookie: soon_view=thumb
Set-Cookie: stay_login=1
Set-Cookie: suggest_view=thumb
Set-Cookie: user="2|1:0|10:1558786932|4:user|44:NzhkMjIyYTY1MDY4NDNhMWE3YjA1YjkyMGUyYTRiZDA=|f8df570b20ab2cfd6919db2f8b1d147d6d53f342284092b97dcc977ba8c1234d"
Set-Cookie: wanted_view=thumb<br><strong>method</strong>: POST<br><strong>remote_ip</strong>: 192.168.1.215<br></p>
                                    <button onclick="window.location='/errorlogs/';">View Log(Errors)</button>

**Comment by WebSpider** [comment]
Also probably related to #5495

**Comment by doucheymcdoucherson** [comment]
fixed in 826eb10
